setup.py

```diff
@@ -234,11 +253,16 @@ def run(self):
                     if len(args) > 3:
                         subprocess.run(args, check=True, stdout=subprocess.PIPE)
 
+                dest = "onnxruntime/capi/libonnxruntime_providers_openvino.so"
+                if path.isfile(dest):
+                    result = subprocess.run(["patchelf", "--set-rpath", "$ORIGIN", dest, "--force-rpath"],
+                                            check=True, stdout=subprocess.PIPE, universal_newlines=True)
+
                 self._rewrite_ld_preload(to_preload)
                 self._rewrite_ld_preload_cuda(to_preload_cuda)
                 self._rewrite_ld_preload_tensorrt(to_preload_tensorrt)
             _bdist_wheel.run(self)
-            if is_manylinux and not disable_auditwheel_repair:
+            if is_manylinux and not disable_auditwheel_repair and not is_openvino:
                 file = glob(path.join(self.dist_dir, "*linux*.whl"))[0]
```

Comment:
self.dist_dir is member of _bdist_wheel which gets its value during runtime. At compilation its set to None. Pylint is throwing error for adding path to dist_dir. Should we set a path to this member variable ?